app/src/main/res/layout/onboarding_manual_signin.xml

```diff
@@ -26,37 +26,31 @@
         android:id="@+id/header_text"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:textAppearance="@style/Header16TextStyle"
+        android:textAppearance="@style/HeaderTextStyle"
         android:lineSpacingExtra="8sp"
-        android:textColor="@color/neutral_text"
-        app:drawableTint="@color/white_color"
+        android:layout_marginTop="10dp"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="@id/avatar_icon"
         android:layout_marginStart="52dp"
-        tools:text="@string/onboarding_account_sign_in_header"
-    />
+        tools:text="@string/onboarding_account_sign_in_header1" />
 
-    <org.mozilla.fenix.utils.LinkTextView
+    <TextView
```

Comment:
I think we need to update all the onboarding description text styles. I am seeing they should be more bold, using Inter font and 14sp. Can you check the styles in figma (you can select the android styles to see their values), I know there's also lineSpacingExtra, but I don't know if we use that value.

Reply:
We have to confirm with UX as we only have been using the `metropolis` font.